src/applications/vaos/new-appointment/components/ReviewPage/PreferredDatesSection.jsx

```diff
@@ -16,15 +25,19 @@ export default function PreferredDatesSection(props) {
             </ul>
           </div>
           <div>
-            <Link
-              to={newAppointmentFlow.requestDateTime.url}
+            <VaLink
```

Comment:
Let's use `va-link` for consistency
```suggestion
            <va-link
```